src/TokenMask/CsrfTokenMask.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Csrf\TokenMask;
+
+use Yiisoft\Security\TokenMask;
+
+final class CsrfTokenMask implements CsrfTokenMaskInterface
```

Comment:
Why do we need this one? There's a correct way to do masking security-wise and I see no reason not to use it directly.

Reply:
For example, you want to add a time interval prefix to the token. This is a way to do this. Another way is `CsrfTokenInterface`

Reply:
That isn't a good idea. In this case we should not give any ability to customize masking at all.

Reply:
Let's remove the interface.